### PR TITLE
bitcoin/formatting: avoid removing trailing zeroes for BTC amounts

### PR DESCRIPTION
**Pull request by Beerosagos**

### CODE REVIEW

File: src/rust/util/src/decimal.rs [hunk 1]
```diff
@@ -32,6 +32,14 @@ pub fn format<F: Format>(value: F, decimals: usize) -> String {
     v.trim_end_matches('0').trim_end_matches('.').into()
 }
 
+pub fn format_no_trim<F: Format>(value: F, decimals: usize) -> String {
```
Comment (benma) [inline src/rust/util/src/decimal.rs:35]: Please add a docstring.

**Review by benma — APPROVED** [review]
Nice!

File: src/rust/util/src/decimal.rs (hunk 1 above)
Comment (Beerosagos) [reply src/rust/util/src/decimal.rs:35]: I knew I forgot something! :|

**Comment by benma** [comment]
@Beerosagos we also forgot the CHANGELOG entry! please add :D

**Comment by Beerosagos** [comment]
> @Beerosagos we also forgot the CHANGELOG entry! please add :D

Sorry :facepalm: made a new PR: https://github.com/digitalbitbox/bitbox02-firmware/pull/1097

**Pull request merged**